src/main/java/Eln.java

```diff
@@ -0,0 +1,47 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    static String LINE = "____________________________________________________________";
+    private static void echoUser() {
+        Scanner scan = new Scanner(System.in);
+        String input = "";
+        String[] list = new String[20];
+        int index = 0;
+
+        while(!input.equals("bye")) {
+            input = scan.nextLine();
+
+            if(input.equals("list")) {
+                System.out.println(LINE);
+                if(index == 0) {
+                    System.out.println("list is empty");
+                }
+                for(int i = 0; i < index; i++) {
+                    System.out.println((i+1) + ". " + list[i]);
+                }
+                System.out.println(LINE);
+            } else {
+                list[index] = input;
+                System.out.println(LINE);
```

Comment:
I think it would be good if you created a private method to handle printing different segments, to avoid typing System.out.println(LINE) many times in your code and to create a level of abstraction.